### DISCUSSION TITLE
ebpf 技术与系统观测

### DISCUSSION THREAD

**Discussion by giscus** [edony-ink/comments#6] [General]
# ebpf 技术与系统观测

bpf 在 observability 领域最大的优势是：数据全面、无损探伤、性能较好、软件定义数据（Sofaware-Defined Data），所以 bpf 技术支持观测场景背后的理念应该是：帮助观测系统更快、更好、无损地获取可编程的观测数据

https://www.edony.ink/ebpf-and-observability-system/

**Comment by edonyzpc** [comment]
bpf 技术作为最新的热点，在观测、安全、网络等领域都有非常多的运用，比较出名有 cilium、ehids、pixie、skywalking 等项目，这篇文章主要关注的是 bpf 在观测领域运用时的产品逻辑和设计哲学
